.github/workflows/bumpVersions.yml

```diff
@@ -4,35 +4,28 @@ name: Bump component versions
 on:
   workflow_dispatch:
```

Comment:
I would think a bit about how we run the updatecli pipeline here, given that:

* we **must** run it once every release, we **should** run it on a schedule
* there is still the yaml reformat issue we should take care manually
* you simplified the apply from the user perspective by moving the loop inside the template

wouldn't be better to follow the same approach we currently have in the ansible repo where we expect the user to run updatecli locally, double check that everything is as expected and then raise the PR?

this is already a giant leap forward from where we started

otherwise LGTM and decide later